node_modules/oae-core/userdetails/js/userdetails.js

```diff
@@ -0,0 +1,161 @@
+                    // show a success notification and close the modal
+                    $('#userdetails-modal', $rootel).modal('unlock');
+                    $('#userdetails-modal', $rootel).modal('hide');
+                    oae.api.util.notification(
+                        oae.api.i18n.translate('__MSG__PROFILE_EDITED__'),
+                        oae.api.i18n.translate('__MSG__PROFILE_DETAILS_EDIT_SUCCESS__', 'userdetails'));
+
+                    // Trigger an event so other elements can be updated
+                    $(document).trigger('oae.editprofile.done', data);
+                });
+            });
+
+            // Avoid default form submit behaviour
+            return false;
+        };
+
+        /**
+         * Hide the user details form and show the Terms and Conditions
+         */
+        var showTermsAndConditions = function() {
+            $('#userdetails-form', $rootel).hide();
+            $('#userdetails-terms-and-conditions-container', $rootel).show();
+            $('.modal-dialog', $rootel).addClass('termsandconditions');
+            $('.modal-footer', $rootel).show();
+        };
+
+        /**
+         * Show the user details form and hide the Terms and Conditions
+         */
+        var showUserDetailsForm = function() {
+            $('#userdetails-form', $rootel).show();
+            $('#userdetails-terms-and-conditions-container', $rootel).hide();
+            $('.modal-dialog', $rootel).removeClass('termsandconditions');
+            $('.modal-footer', $rootel).hide();
+        };
+
+        /**
+         * Add the bindings to show or hide the Terms and Conditions
+         */
+        var addTermsAndConditionsBinding = function() {
+            // Bind the Terms and Conditions link
+            $rootel.on('click', '#register-terms-and-conditions', function() {
+                // Get the Terms and Conditions and render them if we hadn't done so already
+                if ($('#userdetails-terms-and-conditions-container', $rootel).is(':empty')) {
+                    oae.api.user.getTC(function(err, data) {
+                        oae.api.util.template().render($('#userdetails-termsandconditions-template', $rootel), {
+                            'terms': data.text
+                        }, $('#userdetails-terms-and-conditions-container', $rootel));
+                    });
+                }
+
+                // Hide the user details form and show the Terms and Conditions
+                showTermsAndConditions();
+            });
+
+            // Bind the `back` button
+            $rootel.on('click', '#userdetails-back', function() {
+                // Show the user details form and hide the Terms and Conditions
+                showUserDetailsForm();
+
+                // As the Terms and Conditions have now been seen by the user,
+                // we check the Terms and Conditions checkbox
+                $('#userdetails-accept-terms-and-conditions', $rootel).prop('checked', true);
```

Comment:
I don't think we can do this. Just because they read them doesn't mean they've accepted them.